src/urllib3/__init__.py

```diff
@@ -80,3 +82,28 @@ def disable_warnings(category=exceptions.HTTPWarning):
     Helper for quickly disabling all urllib3 warnings.
     """
     warnings.simplefilter("ignore", category)
+
+
+# PoolManager for a top-level request method.
+# DEFAULT_POOL is a module global PoolManager instance,
+# therefore its side effects could be shared across dependencies relying on it.
+# To avoid side effects create a new PoolManager instance and use it instead.
+try:
+    import certifi
+
+    DEFAULT_POOL = PoolManager(cert_reqs="CERT_REQUIRED", ca_certs=certifi.where())
+except ImportError:
+    DEFAULT_POOL = PoolManager()
+
+
+def request(method, url, fields=None, headers=None, **urlopen_kw):
```

Comment:
Shouldn't we omit `**urlopen_kw` in the signature if we're going to drop them on the floor anyways?

Reply:
Right, makes more sense, it is better to get an error when passing keyword arguments instead of having them silently dropped. Removed the ``**urlopen_kw`` from the signature.